db/db_test2.cc

```diff
@@ -849,8 +853,12 @@ TEST_F(DBTest2, WalFilterTestWithChangeBatch) {
       num_keys_added_(0) {}
     void Put(const Slice& key, const Slice& value) override {
       if (num_keys_added_ < num_keys_to_add_in_new_batch_) {
-        new_write_batch_->Put(key, value);
-        ++num_keys_added_;
+        Status s = new_write_batch_->Put(key, value);
+        if (s.ok()) {
```

Comment:
Why not an ASSERT_OK?

Reply:
This is inside a callback and so I wasn't certain that it was safe to use ASSERT_* here.